core/controllers/editor.py

```diff
@@ -790,6 +1199,39 @@ class LearnerAnswerInfoHandler(EditorHandler):
     """Handles the learner answer info for an exploration state."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'entity_type': {
+            'schema': {
+                'type': 'basestring',
+                'choices': [
+                    feconf.ENTITY_TYPE_EXPLORATION,
+                    feconf.ENTITY_TYPE_QUESTION
+                ]
+            }
+        },
+        'entity_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'DELETE': {
+            'state_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'learner_answer_info_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Ditto.

Reply:
Same question, it is raising 404 error, do I have to fix it ?